code/datums/components/deployable_item.dm

```diff
@@ -76,9 +76,12 @@
 		var/newdir = user.dir //Save direction before the doafter for ease of deploy
 		if(!do_after(user, deploy_time, TRUE, item_to_deploy, BUSY_ICON_BUILD))
 			return
-		if(LinkBlocked(get_turf(user), location))
-			location.balloon_alert(user, "No room to deploy")
-			return
```

Comment:
Why are you removing this check? This makes you able to bypass one-per-tile limitations by stacking doafters. Conditions can never be trusted to be the same after doafters.

Reply:
(This is also the culprit on how people are making instaklill spike traps)

Reply:
(This is also the culprit that is allowing people to stack these to make instakill traps)